.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -3,6 +3,10 @@ name: "\U0001F41B Bug Report"
 about: Typo, Grammatical error, etc.。
 ---
 
+<!-- markdownlint-disable MD012 -->
+<!-- markdownlint-disable MD029 -->
+<!-- markdownlint-disable MD041 -->
```

Comment:
I think it would be better to skip `.github` directory in CI.

Reply:
Of course, I will fix this

Reply:
ok, @aylei will fix this.

Reply:
This directory is indeed skipped in the CI

Reply:
He said that currently the markdownlint check does skip `.github` directory now. I will remove these three notes from `.github`.

Reply:
Fixed it in https://github.com/pingcap/docs-cn/pull/1611.